AutoMerge.jl/Project.toml

```
@@ -0,0 +1,62 @@
+name = "AutoMerge"
+uuid = "c5732277-7834-41e3-8e1f-5f375898cfe1"
+authors = ["Dilum Aluthge <dilum@aluthge.com>", "Fredrik Ekre <ekrefredrik@gmail.com>", "contributors"]
+version = "1.0.0"
+
+[deps]
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+GitHub = "bc5e4493-9b4d-5f90-b8aa-2b2bcaad7a26"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+RegistryCI = "0c95cc5f-2f7e-43fe-82dd-79dbcba86b32"
+SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+LicenseCheck = "726dbf0d-6eb6-41af-b36c-cd770e0f00cc"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+RegistryTools = "d1eb7eb1-105f-429d-abf5-b0f65cb9e2c4"
+StringDistances = "88034a9c-02f8-509d-84a9-84ec65e18404"
+TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
+Tar = "a4e569a6-e804-4fa4-b0f3-eef7a1d5b13e"
+TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
+VisualStringDistances = "089bb0c6-1854-47b9-96f7-327dbbe09dca"
+
+[compat]
+Base64 = "<0.0.1, 1"
+Dates = "<0.0.1, 1"
+GitHub = "5.2"
+HTTP = "0.8, 0.9.1, 1"
+JSON = "0.19, 0.20, 0.21"
+LibGit2 = "<0.0.1, 1"
+LicenseCheck = "0.2"
+Pkg = "1"
+Printf = "<0.0.1, 1"
+Random = "<0.0.1, 1"
+RegistryTools = "2.2"
+RegistryCI = "10.0, 11.0"
```

Could be simpler as this?

```suggestion
RegistryCI = "10, 11"
```